final_task/rss_reader/personal_exceptions.py

```diff
@@ -0,0 +1,15 @@
+class IncorrectURL(Exception):
+    def __str__(self) -> str:
+        return 'The entered URL is incorrect'
```

Comment:
You do not really need this logic if you will raise exceptions like this: `raise IncorrectUrl("message")`, `pass` will do